cs-amazon/src/main/java/io/cloudslang/content/amazon/utils/InputsUtil.java

```diff
@@ -496,6 +499,13 @@ private static void setSpecificQueryParamValue(Map<String, String> queryParamsMa
                 currentArray[index], currentArray.length > START_INDEX);
     }
 
+    private static void validateNetmask(String input) {
+        int netmaskIndex = indexOf(input, SCOPE_SEPARATOR) + ONE;
```

Comment:
@victorursan  Well, I did received some complains about readability... otherwise I agree with you ... ONE is 1 and it stays the same... 😃